twitter-app-example.yml

```diff
@@ -0,0 +1,4 @@
+oAuthConsumerKey: <fill in>
```

Comment:
create a parent "Twitter" element and have the fields live underneath there. This is better practice as you could have oauthConsumerSecrets for other networks such as facebook, instagram, etc. If you were to keep this format, it should be labeled with twitter.